pkgs/development/python-modules/nsenter/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, contextlib2, argparse, flake8, pathlib, pythonOlder, pythonAtLeast }:
+
+buildPythonPackage rec {
+  pname = "nsenter";
+  version = "0.2";
+
+  propagatedBuildInputs = [
+    contextlib2
+    argparse
+    flake8
+  ] ++ lib.optional (pythonOlder "3.4") [
+    pathlib
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jr5592pshzn0ck75xb4f8dmcya8vb7x8byp9j7991fy0g5ihsl7";
+  };
+
+  prePatch = lib.optionalString (pythonAtLeast "3.4") ''
+    sed '/^pathlib$/d' -i requirements.txt # included in std since python 3.4
+  '';
+
+  doCheck = false;
```

Comment:
why?

Reply:
Added a comment on why. The tarball from PyPi does not contain any tests and not disabling checkPhase will result in an error.